tests/test_annotations.py

```diff
@@ -60,10 +61,30 @@ def v7_annotations(self):
             self._v7_annotations = WsiAnnotations.from_darwin_json("tests/files/103S.json")
         return self._v7_annotations
 
-    def test_asap_to_geojson(self, split_per_label=False):
-        asap_geojson = self.asap_annotations.as_geojson(split_per_label=split_per_label)
-        geojson_geojson = self.geojson_annotations.as_geojson(split_per_label=split_per_label)
-        assert asap_geojson == geojson_geojson
+    def test_asap_to_geojson(self):
+        # TODO: Make sure that the annotations hit the border of the region.
+        asap_geojson = self.asap_annotations.as_geojson()
+        geojson_geojson = self.geojson_annotations.as_geojson()
+
```

Comment:
could also check that the number of annotations in asap_geojson and geojson_geojson are the same

Reply:
I think the length can be different as you can cut up polygons, you need to check if the polygons are equal (which is done in the last line).

Reply:
okay , if the length of one of the lists would be shorter, the iteration would stop when using zip.

Reply:
so my comment was regarding the asap_geojson and the geojson_geojson.

Reply:
Isn't the test to check if the polygons are the same sufficient?